cache.js

```diff
@@ -8,16 +8,9 @@ const path = require('path')
  * Get the cache path for this PR branch tag
  * @param {number} repoID repository identifier
  * @param {number} prID pull request identifier
- * @param {string} app Optional: an app that calls the function. (default bandit)
- * It's needed because gosec uses GOPATH.
  */
-function getBranchPath (repoID, prID, app) {
-  app = app || 'bandit'
-  if (app === 'bandit') {
-    return path.join('cache', repoID.toString(), prID.toString())
-  } else if (app === 'gosec') {
-    return path.join('cache/go/src', repoID.toString(), prID.toString())
-  }
+function getBranchPath (repoID, prID) {
+  return path.join('cache/go/src', repoID.toString(), prID.toString())
```

Comment:
So it's a little odd that non-Go language files with be added into a go named cache directory.  It would add confusion.  Can we use a path without the name of `Go` in it?

Reply:
Well, no we can't. Here are is the proof:
1) In the Golang documentation here is what is mentioned about the GOPATH environmental variable: https://golang.org/doc/code.html#GOPATH
2) I spoke with Antoine about this before https://github.com/vmware/precaution/pull/56#issuecomment-445218183
3) Still, I tried to change that. I changed the getBranchPAth function:
![image](https://user-images.githubusercontent.com/16246778/52519193-5bcaa300-2c60-11e9-8bfb-b6a575ee215f.png)

and here is the result from the logs from Heroku:
![image](https://user-images.githubusercontent.com/16246778/52519200-78ff7180-2c60-11e9-9de3-194a4b89b0f2.png)